msmtools/analysis/sparse/stationary_vector.py

```diff
@@ -140,18 +140,6 @@ def stationary_distribution(T):
         Vector of stationary probabilities.
 
     """
-    fallback = False
-    try:
-        mu = stationary_distribution_from_backward_iteration(T)
-        if np.any(mu < 0):  # numerical problem, fall back to more robust algorithm.
-            fallback=True
-    except RuntimeError:
-        fallback = True
-
-    if fallback:
-        mu = stationary_distribution_from_eigenvector(T)
-        if np.any(mu < 0):  # still? Then set to 0 and renormalize
```

Comment:
Are you sure you want to remove this and use the abs (above) instead?
It wont result in the same behavior.